src/observation/encodeMatcher/test.js

```diff
@@ -43,16 +56,21 @@ describe('SQLite encodeMatcher', () => {
     })
   })
   it('throws on queries it cannot encode', () => {
-    const error = `can't be encoded into a matcher`
-    expect(() => makeMatcher([Q.on('projects', 'team_id', 'abcdef')])).toThrow(error)
-    expect(() => makeMatcher([Q.experimentalJoinTables(['foo'])])).toThrow(error)
-    expect(() => makeMatcher([Q.experimentalNestedJoin('foo', 'bar')])).toThrow(error)
-    expect(() => makeMatcher([Q.experimentalSortBy('left_column', 'asc')])).toThrow(error)
-    expect(() => makeMatcher([Q.experimentalTake(100)])).toThrow(error)
-    expect(() => makeMatcher([Q.experimentalTake(100)])).toThrow(error)
-    expect(() => makeMatcher([Q.unsafeLokiFilter(() => {})])).toThrow(error)
+    unencodableQueries.forEach(query => {
+      // console.log(query)
```

Comment:
Do you want to keep this? :)

Reply:
I do! if you're running an expect() in a loop and one of those expectations fails, it's usually difficult to figure out which one is it - so i'll then add a clg to check which one failed. So i'm keeping this for DX :)

Reply:
Make sense, just wanted to be sure :)